utils/load_mml.m

```diff
@@ -139,6 +139,8 @@ function insertpvs(index, pvs)
                 field = 'b1';
             elseif strcmp(type, 'SEXT')
                 field = 'b2';
+            elseif strcmp(type, 'VSTR')
+                field = 'a0';
             else
```

Comment:
Yes.  This was a shortcut because everything left was either a bending magnet or a horizontal steerer.